__tests__/unit/git-repo.test.ts

```diff
@@ -37,4 +44,33 @@ describe('GitRepo', () => {
 
     expect(gitRepo.isInitialized()).toBe(true)
   })
+
+  it('should check if a repo has commits', async () => {
```

Comment:
Maybe:
`initialize new git repository, check that there are no comments, add a test comment, check for the comment`